pages/linux/perf.md

```diff
@@ -0,0 +1,19 @@
+# perf
+
+> Framework for linux performance counter measurements.
+
+- Gather basic performance counter stats for a command:
```

Comment:
It also displays the stats in the console right ? So maybe Display is a better word than Gather. Since Gather does not specify whether it will be showed or not.

Reply:
Sure..